### ISSUE TITLE
Do training and testing

### ISSUE THREAD

**Issue by 14corman** [14corman/JMIProject#18]
Once NN.py and preprocess.py is ready, do what I did before with training many models with different learning rates and drop out rates. This time though, forget 1 since it is the same as 0. I will start off with the best layer set achieved by the previous model and see what that looks like at first.

**Comment by 14corman** [comment]
After over 32 hours of training models, my computer finally finished and there are 3 top models that I had to pick from. The charts are given below:
![image](https://user-images.githubusercontent.com/15676385/56129463-c2618b00-5f47-11e9-9c1b-b36b1e9b1ca3.png)

![image](https://user-images.githubusercontent.com/15676385/56129488-d60cf180-5f47-11e9-9ac4-5f1ee6ee8a6f.png)

![image](https://user-images.githubusercontent.com/15676385/56129512-eae98500-5f47-11e9-9195-3210d676e236.png)

![image](https://user-images.githubusercontent.com/15676385/56129536-fd63be80-5f47-11e9-9a87-2ceef8cd3ea2.png)

![image](https://user-images.githubusercontent.com/15676385/56129559-0e143480-5f48-11e9-883d-627f7906fa44.png)

As you can see, the green and orange are very close in both f1 score and accuracy. Both were able to achieve about 0.89 for both. The blue one did not get that far, but if you look at its loss it looks like it was still possibly steadily going down. That means it may not have started overfitting yet and may have reached the point of the other 2. Between the orange and green, the green would have to be chosen as it has just barely better precision than the orange does. With that said, the winning model it looks like based on adding the 2 demographic variables is:

batch norm at each layer
layers - 1000,2000,2000,10000
 drop out - 0.2 
learning rate - 0.001

Most other models learned a little then started to aggressively overfit the data and actually go down in f1 score. These 3 did not reach that point before being cut off.
